changelogs/fragments/513-snmp-resource-module.yaml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
```

Comment:
Does a new resource module count as a "breaking change?" It would not disrupt anything in the previous version after all.